padguidedb/padguidedb.py

```diff
@@ -35,19 +35,15 @@ def __init__(self, bot: Red, *args, **kwargs):
         self.bot = bot
         self.settings = PadGuideDbSettings("padguidedb")
 
-        self.executor = concurrent.futures.ThreadPoolExecutor(max_workers=1)
         self.queue_size = 0
-        self.full_etl_running = False
-        self.extract_images_running = False
-
+        self.full_etl_lock = asyncio.Lock()
+        self.extract_images_lock = asyncio.Lock()
+        self.semaphore = asyncio.BoundedSemaphore(5)
```

Comment:
'semaphore' is a bad name for a variable. There shouldn't be multiple leases anyway since only one job can run at a time.